packages/runtime/src/widgets/Card.tsx

```diff
@@ -41,17 +42,37 @@ export const Card: React.FC<CardProps> = ({ children, styles }) => {
   const renderedChildren = useMemo(() => {
     return EnsembleRuntime.render(children);
   }, [children]);
+
   const mergedStyles = merge(defaultStyles, styles);
-  const { shadowOffset, shadowBlur, shadowSpread, shadowColor } = mergedStyles;
+  const { shadowOffset, shadowBlur, shadowSpread, shadowColor, gap } =
+    mergedStyles;
+
   return (
-    <div
-      style={{
-        ...mergedStyles,
-        boxShadow: `${shadowOffset} ${shadowOffset} ${shadowBlur} ${shadowSpread} ${shadowColor}`,
-      }}
-    >
-      {renderedChildren}
-    </div>
+    <>
+      <style>
+        {`
+          #ensemble-card > :not(:first-child) {
+            ${
+              gap
+                ? `
+                    display: block !important;
+                    margin-top: ${gap} !important;
+                  `
+                : ""
+            }
+          }
+        `}
```

Comment:
Why is this needed?

Reply:
In that case, why not just allow setting the styles on the card element directly?